travis/build-and-test.sh

```diff
@@ -34,3 +34,18 @@ make -j$(nproc)
 
 # run all unit tests
 ctest -V
+
+# install libappimage
+if [[ `whoami` != root ]]
+  then sudo make install
+  else make install
+fi
+
+# do integration test
+pushd $TEMP_BASE
```

Comment:
This change is wrong in this PR. I'd rather discuss it in a second one. JFYI: I'm not a fan of this testing, and I'll explain why there.

Reply:
Test cases and changes related to them should go in the same PR. That's how you make sure that the change you are making is right.

This kind of testing indeed modifies the target system. That's precisely the feature that we must test. That the change made to the target system is the right one. What we could do is split this from the regular unit tests.